public_html/wp-content/mu-plugins/camptix-tweaks/camptix-tweaks.php

```diff
@@ -77,7 +77,7 @@ function show_sandbox_mode_warning() {
 
 		// And the event is not archived
 		if ( $sandboxed && ! $camptix_options['archived'] ) {
-			require_once( __DIR__ . '/views/notice-sandbox-mode.php' );
+			require_once __DIR__ . '/views/notice-sandbox-mode.php';
```

Comment:
Most of the changes in this file are autoFix from linter, the ones that this PR focuses has been highlighted.

Reply:
Ideally, they would be done in a different PR as it will make it harder to trace history.